README.md

```diff
@@ -39,7 +61,7 @@ giving us both centralized and local logs.  (Redundancy is nice).
 
 	import (
 		"flag"
-		"github.com/Graylog2/go-gelf/gelf"
+		gelf "gopkg.in/Graylog2/go-gelf/gelf.v1/gelf"
```

Comment:
The paths used here seem rather different from the [gopkg.in example(https://labix.org/gopkg.in):

    go get gopkg.in/yaml.v1

Especially, we have the part with the version number in the middle, and `gelf` repeated in `gelf.v1/gelf`. What's up with all that?